### DISCUSSION TITLE
Is it possible for you to add a way to import music?

### DISCUSSION THREAD

**Discussion by AditMistry** [z-huang/music#158] [Ideas]
I'd like to permanently move away from Spotify and Amazon Music. If you could add a way to import music from those two sources, that would be awesome!

Thank you for all your hard work on this app! I'm looking forward to more developments!

**Comment by z-huang** [comment]
There's no way to directly migrate to YouTube. But you can import YouTube playlist to the library.

You can see the preview of the next version by install the debug APK from `feature/innertube` branch.

**Reply by Baarten** [reply]
Hey, thx for the nice app!

I can't seem to find the setting or button to import YouTube playlists. Isn't it in build 556?

**Reply by owenconnorz** [reply]
In innertube branch then actions page it's the debug version and to import if u have a playlist on yt music just type your name 1st and last name and it should so your playlist and then gives u the option to import

**Reply by Baarten** [reply]
Ah, thank you. I think that I now know how it's supposed to be done. Unfortunately the search function only works on the _All_ filter for me and my playlist is not there when I search with my first and last name. I verified that the playlist is on YouTube Music and it's public on the account with my first and last name as the account name. When I try to filter for _Community playlists_ to see more than 3 playlists that _All_ shows, it doesn't load and displays a _Retry_ button.
I will wait till `feature/innertube` is merged into `dev` to see if I can find my playlist then.

EDIT: Seems like I can't find it with other YouTube Music apps either. Will look into this later.

**Reply by z-huang** [reply]
Is this #136?

**Comment by owenconnorz** [comment]
Can always try this to transfer spotify to youtube music then import to music debug works well :) 

https://www.tunemymusic.com/

**Reply by Mehtab696** [reply]
use MusConv to move away from Spotify and Amazon Music easily-[http://www.musconv.com/](http://www.musconv.com/?fbclid=IwAR0QABKzQTHviKlRjjUg6WQygPx6i71WDVozP74cuV_H6zQIgk5YzcGdaaE)
